src/protocols/login.js

```diff
@@ -97,38 +98,33 @@ type JwsPayload = {
 };
 
 /**
- * Sign signatureRequest
- * @param {Object} signatureRequest
- * @param {String} idAddr
- * @param {String} ethName
- * @param {Object} proofEthName
+ * Sign a SIGV01 packet with ksign as idAddr
  * @param {Object} kc
+ * @param {String} idAddr
  * @param {String} ksign - public key in hex format
  * @param {Object} proofKSign
- * @param {Number} expirationTime
+ * @param {Number} expirationTimeDelta
+ * @param {Object} data
+ * @param {Object} form
  * @returns {String} signedPacket
  */
-function signIdenAssertV01(signatureRequest: any, idAddr: string,
-  ethName: string, proofAssignName: proofs.ProofClaim, kc: kCont.KeyContainer, ksign: string,
-  proofKSign: proofs.ProofClaim, expirationTime: number): string {
+function signPacket(kc: kCont.KeyContainer, idAddr: string, ksign: string, proofKSign: proofs.ProofClaim,
```

Comment:
This could be generic sign packet creation. It would imply to define a new function for `GENERICSIGV01` a it is done for `IDENASSERTV01` on `signIdenAssertV01`

Reply:
done!